taskcluster/script.py

```diff
@@ -166,6 +165,11 @@ def main():
         print('Android device version (ro.build.version.release):  {}'.format(android_version))
         # this can explode if an unknown device, explode now vs in an hour...
         device_type = get_device_type(device)
+        # set device to UTC
+        device.shell('setprop persist.sys.timezone "UTC"', timeout=ADB_COMMAND_TIMEOUT)
```

Comment:
Don't use shell directly, use shell_output. This didn't work on my p2 without being root so you probably need root=True here.

Reply:
Thanks. Fixed.